app/experimenter/experiments/bugzilla.py

```diff
@@ -38,13 +53,52 @@ def format_bug_body(experiment):
             ]
         )
         bug_body = experiment.BUGZILLA_PREF_TEMPLATE.format(
-            experiment=experiment, variants=variants_body
+            experiment=experiment,
+            variants=variants_body,
+            countries=countries,
+            locales=locales,
         )
 
     return bug_body
 
 
-def make_bugzilla_call(url, data):
+def format_update_body(experiment):
+    cf_tracking = "cf_tracking_firefox{}".format(
+        get_firefox_major_version(experiment.firefox_version)
+    )
+    summary = "[Experiment] {experiment_name} Fx {version} {channel}".format(
+        experiment_name=experiment,
+        version=experiment.firefox_version,
+        channel=experiment.firefox_channel,
+    )
+    return {
+        "summary": summary,
+        "cf_user_story": format_bug_body(experiment),
+        "whiteboard": experiment.STATUS_SHIP_LABEL,
+        cf_tracking: "?",
+    }
+
+
+def update_experiment_bug(experiment):
+    body = format_update_body(experiment)
+    make_bugzilla_put_call(
+        settings.BUGZILLA_UPDATE_URL.format(id=experiment.bugzilla_id), body
+    )
+
+
+def make_bugzilla_put_call(url, data):
```

Comment:
Also if the only difference here is `requests.put` instead of `requests.post` then just collapse it back into a single function and make that an argument like

    def make_bugzilla_call(url, data, method=requests.post):
        try:
            response = method(url, data)
        ....

    make_bugzilla_call(url, data, method=requests.put)

Reply:
kinda did this^ but left the default out since for tests it needed to have the request method stated instead of defaulting to post or else it would use the requests instead of the mock.
Also had to mock out "put" and "post" since mocking requests made requestExceptions funny